source/management_console/index.js

```diff
@@ -7,6 +7,7 @@
 var fs = require('fs');
 var toml = require('toml');
 var url = require('url');
+const CryptoJS = require('crypto-js');
```

Comment:
In 'node' environment, you can use built-in 'crypto' module directly instead of external 'crypto-js' module.